packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,138 @@ void main() {
           ));
           handle.dispose();
         });
+
+        testWidgets('ReorderableListView still scrolls when physics property is set', (WidgetTester tester) async {
+          final List<Widget> widgetList = <Widget>[
+            Container(key: const Key('1'), child: const Icon(Icons.account_circle)),
+          ];
+
+          final ScrollConfiguration configuration = ScrollConfiguration(
+           behavior: const ScrollBehavior(),
+           child: ReorderableListView(
+                key: const Key('list'),
+                physics: const AlwaysScrollableScrollPhysics(),
+                onReorder: (int oldIndex, int newIndex) {},
+                children: widgetList,
+                scrollController: ScrollController(),
+          ));
+
+          await tester.pumpWidget(MaterialApp(
+          home: SizedBox(
+            height: 100,
+            width: 100,
+            child: configuration,
+          ),
+        ));
+
+          final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await gesture.moveBy(_kGestureOffset);
+          // Move back to original position.
+          await gesture.moveBy(Offset(0, -_kGestureOffset.dy));
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          final TestGesture overscrollGesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await overscrollGesture.moveBy(_kGestureOffset);
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          ScrollPhysics physics;
+          await tester.pumpWidget(
+            Builder(
+              builder: (BuildContext context) {
+                physics = configuration.behavior.getScrollPhysics(context);
+                return Container();
+              },
+            ),
+          );
+          expect(physics, isNotNull);
+        });
+
+        testWidgets('Default ReorderableListViews are not always scrollable', (WidgetTester tester) async {
+          final ScrollConfiguration configuration = ScrollConfiguration(
```

Comment:
Never mind, I now see that you were trying to use a `ScrollConfiguration` parent to "set" the default platform physics behavior. Although, I'm not sure if this is the right way to approach testing this behavior, since in a real Flutter app, it's unlikely that we'll wrap all `ReorderableListView`s with a `ScrollConfiguration`.